app/controllers/TrusteeAUKCitizenController.scala

```diff
@@ -84,4 +91,16 @@ class TrusteeAUKCitizenController @Inject()(
         }
       )
   }
+
+  private def getMessagePrefix(index: Int, request: DataRequest[AnyContent]) = {
+    val isLead = request.userAnswers.get(IsThisLeadTrusteePage(index)).get
```

Comment:
@jassalrichy Same comment as above regarding the `.get`

Reply:
requiredAnswer check added